server/database.go

```diff
@@ -960,6 +954,10 @@ func (s *Server) composeDatabaseListByFind(ctx context.Context, find *api.Databa
 
 	var dbList []*api.Database
 	for _, dbRaw := range dbRawList {
+		if !find.IncludeAllDatabase && dbRaw.Name == api.AllDatabaseName {
```

Comment:
I think we've already done the filter clausing in the store.
https://github.com/bytebase/bytebase/blob/main/store/database.go#L271